json-reformat.el

```diff
@@ -47,7 +48,7 @@
 ;;; Code:
 
 (require 'json)
-(eval-when-compile (require 'cl))
+(eval-when-compile (require 'cl-lib))
```

Comment:
In principle, this could be reduced to `(require 'cl-macs)`, because only one macro is used. But it probably doesn't matter. ;-)

Reply:
It's actually better _not_ to require `cl-macs`: it isn't present in the separate `cl-lib` backwards-compatibility library on ELPA, so referring to `cl-macs` makes code less portable. Just using `cl-lib` is the right way to go.